packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -418,7 +500,7 @@
             padding: 1.6rem;
             display: flex;
             justify-content: flex-end;
-            border-top: 1px solid var(--general-section-1);
+            border-top: 0.1rem solid var(--general-section-1);
```

Comment:
Why are we using rem here?

Reply:
changed to px back since its all around the app